### PR TITLE
Fix error message change issue from libvirt

### PR DESCRIPTION
**Pull request by chunfuwen**
Error Message update from 'Connection reset by peer'
to 'Could not create file: Permission denied'
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
@kylazhang 
@balamuruhans 
Minor fix, could you help take a look?

Thanks!

File: libvirt/tests/src/virtual_disks/virtual_disks_ceph.py [hunk 1]
```diff
@@ -480,7 +480,8 @@ def clean_up_volume_snapshots():
     if test_snapshot:
         unsupported_err.append('live disk snapshot not supported')
     if test_disk_readonly:
-        unsupported_err.append('Unable to read from monitor: Connection reset by peer')
+        unsupported_err.append('unable to execute QEMU command \'transaction\': ' +
+                               'Could not create file: Permission denied')
```
Comment (balamuruhans) [inline libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: \+ is not required, 
```python
unsupported_err.append('unable to execute QEMU command \'transaction\': ' 
                       'Could not create file: Permission denied')
```

Comment (chunfuwen) [inline libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: remove one line

**Comment by chunfuwen** [comment]
@balamuruhans 

New patch uploaded

**Comment by balamuruhans** [comment]
@chunfuwen please rebase the PR, let us merge it.

**Pull request merged**